backend/src/test/java/ca/gov/dtsstn/cdcp/api/service/UserServiceTests.java

```diff
@@ -85,4 +88,34 @@ void testCreateConfirmationCodeForUser() {
 		assertThat(confirmationCode.getExpiryDate()).isAfter(Instant.now());
 	}
 
+	@Test
+	@DisplayName("Test userService.verifyConfirmationCode(..) with valid input")
+	void testVerifyConfirmationCode_Valid() {
+		final var creationDate = LocalDateTime.now().minusDays(73).toInstant(ZoneOffset.UTC);
+		final var expiryDate = LocalDateTime.now().plusDays(288).toInstant(ZoneOffset.UTC);
+		final var codeValue = "code value";
+		final var confirmationCode = ImmutableConfirmationCode.builder().code(codeValue).createdDate(creationDate).expiryDate(expiryDate).build();
+		final var user = ImmutableUser.builder().addConfirmationCodes(confirmationCode).build();
+
+		assertThat(userService.verifyConfirmationCode(codeValue, user)).isTrue();
+	}
+
+	@Test
+	@DisplayName("Test userService.verifyConfirmationCode(..) with no code")
+	void testVerifyConfirmationCode_NullCodeValue() {
+		final var user = ImmutableUser.builder().build();
+		assertThrowsExactly(IllegalArgumentException.class, () -> userService.verifyConfirmationCode("", user), "code is required; it must not be null");
```

Comment:
`""` is not the same as `null` so this test isn't doing what the method name suggests it is

Reply:
Makes sense, tests have been expanded in commit 2cebd2d6f0685660fa512c66c4a5da16869070e1.